tests/tests_app/utilities/packaging/test_lightning_utils.py

```diff
@@ -9,17 +10,17 @@
     _verify_lightning_version,
 )
 
-# TODO: this is very sensitive test, need to be done
```

Comment:
why this is removed?

Reply:
Because this isn't a TODO, this is more of a note. Didn't bring much.

Reply:
my bad... read it in the reverse direction...